.github/workflows/plugins-ci.yml

```diff
@@ -130,7 +146,7 @@ jobs:
         run: npm i --ignore-scripts
 
       - name: Run tests
-        run: npm test
+        run: NODE_OPTIONS="${{ steps.node-flags.outputs.flags }}" npm test
```

Comment:
I would expect this as a different job, not as an edit of this one.

Reply:
I didn't understand. Since it is in a matrix config, it will run 2 different jobs. 1 for false value, and one for true. (or only 1 if it's not enabled)